### DISCUSSION TITLE
Octernship S23 Results - Congratulations to all the Octerns! 😃

### DISCUSSION THREAD

**Discussion by OmPrakash95** [DiveHQ/backend-internship-task#51] [Announcements]
Hola Hackers 👋🏼

First of all, congratulations to all the selected candidates for the Octernship! 🎉 Github has sent a separate email to your contact email address with the next steps and we look forward to Onboarding y’all.

Last few weeks has been a lot of learning for our team at Dive. We happened to meet a 100s of smart, talented folks across the globe evaluating their work. Our team spent Week's hours in evaluation  (Sorry about the delay this caused) to ensure fair evaluation of each and every submission. 

We had an incredibly talented group of applicants vying for a limited number of spots. To those who secured their place in the Octernship, your success is a testament to your skills, passion, and unwavering commitment. As you embark on this incredible journey, know that you have our utmost support, and we can't wait to witness the remarkable contributions you'll make during and after the Octernship.

To those who gave it their best shot but couldn't make it this time, remember that your journey doesn't end here. The quality of submissions was truly outstanding, and we sincerely appreciate the time and energy you invested in the application process. While this might feel like a setback, I urge you to keep pushing forward, learning, and growing. Opportunities are boundless, and your determination will undoubtedly lead you to success in the future. I encourage you all to consider applying for any [open roles](https://www.notion.so/letsdive/Careers-Dive-c75d4d1ee8c9486fb71b6b4159105353) that align with your interests and expertise in the days to come.

Thank you, everyone, for your participation, enthusiasm, and dedication. We look forward to witnessing your continued growth and success.

—

Dive Engineering

**Comment by imsandeepk** [comment]
But the PR for the feedback is still not closed. :,(
